internal/security/secretstoreclient/common.go

```diff
@@ -26,6 +26,8 @@ import (
 	"net/url"
 )
 
+const NoAuthToken = "none"
```

Comment:
Should this have public global scope?

Reply:
Nope, good catch. Only used in that package... Fixed and also commented why it is needed.